dev/vfs_name.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "serialize_result_type.h"
+#include "functional/platform_definitions.h"
```

Comment:
Oh, I meant that this header file should be #included in _config.h_ so that it is universally available.